app/src/pages/inside/projectSettingsPageContainer/projectSettingsPageContainer.jsx

```diff
@@ -156,9 +158,13 @@ export const ProjectSettingsPageContainer = () => {
   }, [accountRole, extensionsConfig, createTabLink, userRole]);
 
   const navigation = useMemo(() => {
-    const title = <FormattedMessage id="SettingsPage.title" defaultMessage="Project Settings" />;
-    return <Navigation items={config} title={title} />;
-  }, [config]);
+    if (rawPage) {
```

Comment:
May be call it prop as `subPage`? Then we can specify in the URL what kind of sub page we need to see.
So it will be f.e.:
`settings/pluginPage?subPage=pluginSubPageName`